packages/orchestrator-ui-components/src/contexts/OrchestratorConfigContext.tsx

```diff
@@ -17,6 +17,8 @@ export const OrchestratorConfigContext = createContext<OrchestratorConfig>({
     authActive: true,
     useWebSockets: false,
     useThemeToggle: false,
+    showWorkflowInformationLink: false,
+    workflowInformationLinkUrl: '',
```

Comment:
These new props of the config object do not seem to be read from any ENV vars? Looks like you might have forgotten that?

Reply:
https://github.com/workfloworchestrator/example-orchestrator-ui/pull/25

Done

Reply:
Oh yeah part or the app, not the library. Apologies.